### PR TITLE
Removed support for MozReview

### LINKED ISSUE #47: Remove support for MozReview

### PR DESCRIPTION
**Pull request by arshadkazmi42**
MozReview support removed.

- Deleted `mozreview` file
- Removed all references to mozreivew

Closes #47

### CODE REVIEW

**Review by marco-c — CHANGES_REQUESTED** [review]
Thanks! The patch looks good overall, I think you got almost all the instances of MozReview.
The only missing one is the actual MozReview website, I forgot to mention it: reviewboard-hg.mozilla.org. All references to this website should be removed too.

Don't worry about the linting failures, they are preexisting.

**Comment by arshadkazmi42** [comment]
@marco-c Thank you for the review. I have completed the requested changes.

File: manifest.json [hunk 1]
```diff
@@ -28,7 +28,7 @@
     },
     {
```
Comment (marco-c) [inline manifest.json]: This entry should be removed too, we don't want to inject scripts into reviewboard.mozilla.org anymore

Comment (arshadkazmi42) [reply manifest.json]: @marco-c whole object from line 29 - 34?

Comment (marco-c) [reply manifest.json]: Yes, the object referring to mozreview

**Review by marco-c — APPROVED** [review]
Thanks! Everything should have been caught now!

File: manifest.json (hunk 1 above)
Comment (arshadkazmi42) [reply manifest.json]: done with that change

**Comment by arshadkazmi42** [comment]
@marco-c Thank you

**Pull request merged**